wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -523,6 +545,96 @@ public Set<String> getArchitectures() {
             return allArchitectures;
         }
 
+        /**
+         * Returns the default value for the number of positions that the model is capable of
+         * handling. This is calculated as the minimum of the maximum number of positions that the
+         * model can handle (as specified by the model's configuration) and 4096.
+         *
+         * @return the default value for the number of positions that the model is capable of
+         *     handling
+         */
+        public Integer getDefaultNPositions() {
```

Comment:
```suggestion
        public int getDefaultNPositions() {
```

Reply:
This one we want to keep since we use the toString method directly into setting the property as a string.